.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 3
```

Comment:
Could we use 7, instead, for parity with the days until stale? 7 is a bit long for closing, but 3 seems to short so I'm leaning toward the longer range here.

Reply:
Days until close being 3 here means it will be 3 days after being stale ie 10 days in total.

Reply:
I actually meant that we should use 14 days in total @jameesjohn. My concern is that contributors only have 3 days to follow up after the initial comment that the PR is stale to provide changes before the PR is closed.

Reply:
Done.